docker-compose.yml

```diff
@@ -129,13 +129,13 @@ services:
         volumes:
           - ddagent:/tmp/ddagent:rw
     testagent:
-        image: ghcr.io/datadog/dd-apm-test-agent/ddapm-test-agent:v1.11.0
+        image: docker.io/williamconti549/dd-apm-test-agent:integration-versions
         ports:
             - "127.0.0.1:9126:8126"
         volumes:
             - ./tests/snapshots:/snapshots
         environment:
-            - LOG_LEVEL=WARNING
+            - LOG_LEVEL=ERROR
```

Comment:
Why do we need to change the log level? What warnings are we seeing?

Reply:
Reduce noise during debugging but will be switched back